packages/docusaurus/src/client/serverEntry.js

```diff
@@ -140,27 +140,12 @@ async function doRender(locals) {
       useShortDoctype: true,
       minifyJS: true,
     });
-  }
-
-  // TODO this is a temporary error affecting only monorepos due to Terser 5 (async) being used by html-minifier-terser,
```

Comment:
Do I understand correctly that this upgrade fixes it?

Reply:
Thanks, I think it fixes it, not 100% sure though because we still have https://github.com/jantimon/html-webpack-plugin using older versions

Going to assume it fixes it, and opened an issue there for follow-up: https://github.com/jantimon/html-webpack-plugin/issues/1687